app/controllers/admin/invoices_controller.rb

```diff
@@ -10,6 +12,12 @@ def regenerate
       head(:ok)
     end
 
+    def show
+      service = ::Invoices::GeneratePdfService.new(invoice:)
+
+      render html: service.render_html
```

Comment:
Style/MethodCallWithArgsParentheses: Use parentheses for method calls with arguments. (https://rubystyle.guide#method-invocation-parens)